package.json

```diff
@@ -47,7 +47,6 @@
     "bootstrap": "^4.3.1",
     "emotion": "^9.1.3",
     "formulajs": "^1.0.8",
-    "jquery": "^1.9.1",
     "lodash": "^4.17.15",
```

Comment:
gh marks it as a security vulnerability and we don't really need it

Reply:
haha what did we use it for originally?

Reply:
no idea, probs something we did in the very beginning to manipulate the DOM